addons/mail/static/src/core/common/chat_bubble.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+ <templates xml:space="preserve">
+     <t t-name="mail.ChatBubble">
+         <div class="position-relative o-mail-ChatBubble" t-att-name="props.bubble.thread.displayName" t-on-mouseenter="toggleShowPreview" t-on-mouseleave="toggleShowPreview" t-on-click="onClickOpen">
+             <ChatBubblePreview thread="props.bubble.thread" showPreview="state.showPreview" />
+             <div t-if="props.bubble.thread.importantCounter > 0" class="o-mail-ChatBubble-counter position-absolute badge rounded-pill fw-bold o-discuss-badge">
+                 <t t-out="props.bubble.thread.importantCounter"/>
+             </div>
+             <button class="o-mail-ChatBubble-close rounded-pill fw-bold fa fa-close bg-view" t-on-click="onClickClose"/>
+             <ImStatus t-if="props.bubble.thread.correspondent and props.bubble.thread.correspondent.im_status != 'offline'" className="'o-mail-ChatBubble-status'" persona="props.bubble.thread.correspondent" thread="props.bubble.thread"/>
+             <div class="btn o-mail-ChatbubbleContainer-btn shadow">
+                 <img class="o-mail-ChatBubble-avatar" t-att-src="props.bubble.thread.avatarUrl" alt="Thread image"/>
+             </div>
+         </div>
+     </t>
+
+     <t t-name="mail.ChatBubblePreview">
+         <Transition visible="!!(props.showPreview and previewContent)" name="'o-mail-ChatBubble-fade-x'" t-slot-scope="transition">
+             <div t-att-class="{ 'o-mail-ChatBubble-fade-x-out': transition.className.includes('o-mail-ChatBubble-fade-x-leave') }" class="bg-view o-mail-ChatBubblePreview p-1 ms-1 o-mail-ChatBubble-fade-x-in">
```

Comment:
not sure if className.includes() is a good way to do that, seems a little bit unreliable for the string ( any typo then fail )